doc/manual.asciidoc

```diff
@@ -279,6 +279,19 @@ http://clang.llvm.org/docs/JSONCompilationDatabase.html[JSON format] expected
 by the Clang tooling interface.
 _Available since Ninja 1.2._
 
+`missingdeps`:: given a list of targets, look for targets that depend on
```

Comment:
You should add an "Available since Ninja 1.x" line to this, see how it's done above.